kebechet/managers/update/update.py

```diff
@@ -798,6 +798,13 @@ def run(self, labels: list) -> typing.Optional[dict]:
                 comment=ISSUE_CLOSE_COMMENT.format(sha=self.sha),
             )
 
+            update_issue = self.sm.get_issue(_UPDATE_MERGE_REQUEST_TITLE)
+            if update_issue is not None and update_issue.status == 2:  # Means "open" in OGR.
```

Comment:
This is not nice, sadly OGR does not provide any nice alternative and we do not provide any wrapper for this:

https://github.com/packit/ogr/blob/320ba0abd0e76312e9c19db275bc74cb04124560/ogr/abstract.py#L133-L136